ui/desktop/app/styles/app.scss

```diff
@@ -530,3 +530,110 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     background-color: var(--token-color-palette-neutral-50);
   }
 }
+
+.session-details {
+  .rose-layout-body-content {
+    margin-top: 1.5rem;
```

Comment:
Is this `margin-top` something speicifc to session-details? or could be applied to targets too?

Reply:
Both screens need it. I updated the class name to not be specific to just sessions.